packages/actor-rdf-metadata-extract-void/lib/VoidDataset.ts

```diff
@@ -0,0 +1,426 @@
+import type { IQueryEngine, IDataset, QueryResultCardinality } from '@comunica/types';
+import type * as RDF from '@rdfjs/types';
+import { LRUCache } from 'lru-cache';
+import { termToString } from 'rdf-string-ttl';
+import { Algebra } from 'sparqlalgebrajs';
+
+export class VoidDataset implements IDataset {
+  private readonly store: RDF.Store;
+  private readonly identifier: RDF.BlankNode | RDF.NamedNode;
+  private readonly queryEngine: IQueryEngine;
+  private readonly bindingsCache: LRUCache<string, RDF.Bindings[]>;
+
+  public readonly triples: number;
+  public readonly source: string;
+  public readonly vocabularies?: string[];
+  public readonly resourceUriPattern?: RegExp;
+
+  public static readonly RDF_TYPE = 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type';
+  public static readonly RDFS_LABEL = 'http://www.w3.org/2000/01/rdf-schema#label';
+
+  public get uri(): string {
+    return this.identifier.value;
+  }
+
+  public constructor(args: IVoidDatasetArgs) {
+    this.triples = args.triples;
+    this.source = args.source;
+    this.store = args.store;
+    this.identifier = args.identifier;
+    this.vocabularies = args.vocabularies;
+    this.queryEngine = args.queryEngine;
+    this.resourceUriPattern = args.resourceUriPattern;
+    this.bindingsCache = new LRUCache({ max: args.bindingsCacheSize });
```

Comment:
Could you double-check my earlier comment regarding these variable names? (would also apply to the actor)